doc/doc.pro

```diff
@@ -12,5 +12,4 @@ OTHER_FILES += \
     $$PWD/sailfish-webview-webview.qdoc \
     $$PWD/sailfish-webview-webviewflickable.qdoc \
     $$PWD/sailfish-webview-webviewpage.qdoc \
-    $$PWD/sailfish-webview-rawwebview.qdoc \
```

Comment:
See also doc/index.qdoc:47:\li \l{RawWebView}

Reply:
Thanks; I missed that one. Now removed.